OpenMEEG/src/geometry.cpp

```diff
@@ -385,6 +385,26 @@ namespace OpenMEEG {
         return OK;
     }
 
+    bool Geometry::check(const Matrix& mat) const {
+        bool OK = true;
```

Comment:
We should check the size of `mat`, maybe assert there are at least 3 columns.

Reply:
this is taken Care by the `Matrix::operator()`